### DISCUSSION TITLE
只能说非常简陋，非常难用，

### DISCUSSION THREAD

**Discussion by YinGithub** [v2rayA/v2rayA#1416] [General]
有没有更好用的ubuntu client 推荐

**Comment by rigdoge** [comment]
我感觉超好用，便捷，稳定
就是现在更新到最新版，不知道怎么回事，不能插入订阅链接了

**Reply by boyzwj** [reply]
go写的破烂玩意儿。。。我都打算自己开发一个了。。

**Reply by pooneyy** [reply]
> go写的破烂玩意儿。。。我都打算自己开发一个了。。

开发好了能否传到github，现在这个连HY2都不支持，很多节点都用不了

**Reply by bupt2017** [reply]
> > go写的破烂玩意儿。。。我都打算自己开发一个了。。
> 
> 开发好了能否传到github，现在这个连HY2都不支持，很多节点都用不了

你这一问对层主乃是绝杀
